tests/MiniProfiler.Tests/BaseTest.cs

```diff
@@ -173,7 +173,7 @@ protected void AssertPublicPropertiesAreEqual<T>(T t1, T t2)
             // we'll handle any collections elsewhere
             var props = from p in typeof(T).GetProperties(BindingFlags.Public | BindingFlags.Instance)
                         where p.IsDefined(typeof(System.Runtime.Serialization.DataMemberAttribute), false)
-                        && !p.PropertyType.GetInterfaces().Any(i => i.Equals(typeof(IDictionary)) || i.Equals(typeof(IList)))
+                        && !p.PropertyType.GetInterfaces().Any(i => i.Equals(typeof(IEnumerable)))
```

Comment:
This was explicit because, as an example, strings are `IEnumerable`. This would need to change back to the previous behavior.